src/platform/intel/cavs/pm_runtime.c

```diff
@@ -40,8 +40,8 @@
 #include <platform/pm_runtime.h>
 #include <platform/dai.h>
 
-#if defined(CONFIG_APOLLOLAKE)
-//TODO: add support or at least stub api for Cannonlake & Icelake
+#if defined(CONFIG_APOLLOLAKE) || defined(CONFIG_CANNONLAKE)
```

Comment:
SUECREEK ?

Reply:
SueCreek needs to be enabled separately (as has a separate platform directory) but it can be directly based on CannonLake work.